src/IceRpc.Protobuf.Generators/Internal/Parser.cs

```diff
@@ -190,12 +190,30 @@ private IReadOnlyList<ServiceMethod> GetServiceMethods(INamedTypeSymbol interfac
                 items.Length == 1,
                 "Unexpected number of arguments in attribute constructor.");
             string operationName = (string)items[0].Value!;
+            // An IAsyncEnumerable input parameter denotes a client streaming RPC.
+            string inputTypeName = GetFullName(method.Parameters[0].Type);
+            bool isClientStreaming = inputTypeName == "System.Collections.Generic.IAsyncEnumerable";
```

Comment:
You can't compare Type objects/properties instead of this string comparison?

Reply:
I think we can compare the symbols yes, we already do that for the attributes